pootle/core/browser.py

```diff
@@ -180,10 +176,18 @@ def get_children(directory):
     The elements of the list are dictionaries which keys are populated after
     in the templates.
     """
-    directories = [make_directory_item(child_dir)
-                   for child_dir in directory.child_dirs.live().iterator()]
-
-    stores = [make_store_item(child_store)
-              for child_store in directory.child_stores.live().iterator()]
+    directories = []
+    vfti_pks = vftis_for_children(directory).values_list("pk", flat=True)
+    directories = [
+        make_directory_item(
+            child_dir,
+            sort=("priority" if child_dir.pk in vfti_pks else ""))
```

Comment:
This won't work as expected. `child_dir` is a `Directory` while `vftis_for_children()` return `VirtualFolderTreeItem` objects, so a `child_dir.pk` will never be in `vfti_pks`.